src/modules/tests/movieCounter.test.js

```diff
@@ -0,0 +1,33 @@
+/**
+ * @jest-environment jsdom
+ */
+// import { movieCounter } from '../movieCounter.js';
+
+const movieCounter = (movies) => {
+  if (movies.length === 0) {
+    return 'No movies';
+  }
+  if (movies.length > 999) {
+    return '999+';
+  }
+  return movies.length.toString();
+};
+
+describe('movieCounter function', () => {
+  it('should return "No movies" if movies array is empty', () => {
+    const result = movieCounter([]);
```

Comment:
- [ ] As requested earlier, I think it would be important to include tests for your `comments counter`  function to ensure that the app behaves as intended and provides accurate feedback to the user